blocks/cards-carousel/cards-carousel.js

```diff
@@ -109,15 +121,57 @@ function buildSlide(slide, index) {
   return slide;
 }
 
-export default function decorate(block) {
-  decorateIcons(block);
+async function getFeaturesArray() {
```

Comment:
Let's move this to a common location because two other blocks will need the same functionality.
once you get the featuresArray please save it to window.featuresArray so all blocks will first check the window.featuresArray object first and if it doesn't exist then we can call the getFeaturesArray() method.

Reply:
fixed